packages/web-frontend/src/components/View/MatrixWrapper/MatrixWrapper.js

```diff
@@ -296,8 +305,9 @@ export class MatrixWrapper extends Component {
   }
 
   render() {
-    const { isEnlarged, isExporting } = this.props;
-
+    const { isEnlarged, isExporting, viewContent } = this.props;
+    const { presentationOptions, categoryPresentationOptions } = viewContent;
+    const matrixPlaceholder = getPlaceholder(presentationOptions, categoryPresentationOptions);
```

Comment:
I think it's more the responsibility of the function to work out which parts of viewContent it needs - those two aren't otherwise needed or known about within this outer render function, so it's too detailed for its scope.
```suggestion
    const matrixPlaceholder = getPlaceholder(viewContent);
```